comma.go

```diff
@@ -32,12 +28,28 @@ func Comma(v int64) string {
 		j--
 	}
 	parts[j] = strconv.Itoa(int(v))
-	return sign + strings.Join(parts[j:len(parts)], ",")
+	return sign + strings.Join(parts[j:len(parts)], separator)
+}
+
+// Comma produces a string form of the given number in base 10 with
+// commas after every three orders of magnitude.
+//
+// e.g. Comma(834142) -> 834,142
+func (h *EnglishHumanizer) Comma(v int64) string {
```

Comment:
Of course, the choice of a comma is English-centric. The more general concept is the ["decimal mark"](https://en.wikipedia.org/wiki/Decimal_mark) or "decimal separator". [Many locales use a period instead.](https://en.wikipedia.org/wiki/Decimal_mark#Countries_using_Arabic_numerals_with_decimal_point) From there it gets [even weirder](https://en.wikipedia.org/wiki/Decimal_mark#Examples_of_use).